### ISSUE TITLE
Error: Unknown stateMachine with name

### ISSUE THREAD

**Issue by pmotyka** [wmfs/statebox#25]
I'm trying to use the code snippet in the Usage section of the README, but am running in to a few issues.  The error I can't get passed is noted in the following gist: https://gist.github.com/pmotyka/5d0eaf8ec4c2d24a56c975b524e6b096.  Here's the slightly reworked code snippet from the Usage section of the README: https://gist.github.com/pmotyka/bb5e70a8a9b08b1bcc5294775ba8ef2e.  Seems like the issue may be DAO related since the statebox instance appears to have no state machines after being initialized.  Thanks in advance for the guidance!

**Comment by pmotyka** [comment]
Debugging this a bit, there use to be a this.ready.then(() => prior to executing a state machine.  Based on the output in the gist (https://gist.github.com/pmotyka/5d0eaf8ec4c2d24a56c975b524e6b096), it seems like the state execution is being done before statebox is ready.

**Comment by pmotyka** [comment]
I'm still struggling with this issue, any tips on how to work around it?  Worth mentioning, if I switch to statebox@0.0.8 (pre-refactor?), things work fine (output: https://gist.github.com/pmotyka/e3b985a94bd4e879ac6f0dd0583696c8).

**Comment by pmotyka** [comment]
Observation when comparing the output between statebox@0.0.8 and @wmfs/statebox@1.3.2.  With  @wmfs/statebox@1.3.2, there is output 'statebox Using built in MemoryDao' **after** the creation of the state machines.  This suggests the the DAO is being initialized after the state machines wiping out what had been loaded.  Here's a gist with the two outputs, statebox@0.0.8 first, @wmfs/statebox@1.3.2 second: https://gist.github.com/pmotyka/7aad08fbad369127847f9f98b6bb46f9

**Comment by pmotyka** [comment]
I believe I have sorted this out.  It seems that Statebox.createStateMachines(...) returns before it's completed the creation of the state machines.  As a work around, I've wrapped the Statebox.startExecution(...) in a function and invoked it via the callback passed in to Statebox.createStateMachines(...), which appears to file when Statebox.createStateMachines(...) is complete.  Is this working as intended?  It appears statebox@0.0.8 has different behavior, so perhaps a bug introduced in refactoring.  See the following example of the workaround: https://github.com/pmotyka/stateboxexample/blob/master/stateboxexample.js
